### PR TITLE
Menu

### PR DESCRIPTION
**Pull request by andrewjohnso7**

### CODE REVIEW

File: client/src/components/menu/Menu.jsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Dish from './Dish.jsx';
+
+
+const Menu = (props) => {
+  return (
+    <div>Menu
+      {props.dishes.map((dish, index) => (
+        <Dish
```
Comment (kleekich) [inline client/src/components/menu/Menu.jsx]: Are you going to add some more props to pass over later? Is there any reason you did not make this element one line?

**Comment by kleekich** [comment]
@andrewjohnso7 Other than the one React Component being instantiated with multiple line, it looks good to me!

File: client/src/components/menu/Menu.jsx (hunk 1 above)
Comment (andrewjohnso7) [reply client/src/components/menu/Menu.jsx]: I'll make that change for sure

Comment (kleekich) [reply client/src/components/menu/Menu.jsx]: I saw your code has changed. Thanks!

**Pull request merged**